brazilian portuguese.ini

```
@@ -0,0 +1,666 @@
+ShootingMissiles = "Disparando Mísseis"
+ShootingMissiles_Text1 = "Pressione o | pra selecionar Mísseis"
+ShootingMissiles_Text2 = "Segure o | pra ativar Mísseis"
+Autoad = "Autoad"
+Autoad_Text = "Segure o | - Pulo / | - Agarrar"
+
+[ScanEvents]
+ScanEnvironment = "Escaneando Ambiente"
+ScanBioform = "Escaneando Bio-forma"
+ScanMechanism = "Escaneando Mecanismo"
+NewLog = "Nova Entrada do Registro"
+NewLog_Text = "Pressione o | pra abrir o Livro de Registros"
+UpdateLog = "Entrada do Registro Atualizada"
+UpdateLog_Text = "Pressione o | pra abrir o Livro de Registros"
+UpdateLog_Start = "Iniciar"
+Monsters = "Scan do Modelo de Energia"
+Monsters_Text = "Metroids detectados na área"
+
+[SaveLocation]
+Save00 = "Local da Aterrissagem"
+Save01 = "Caverna da Superfície"
+Save02 = "Caverna da Superfície"
+Save03 = "Locais de Procriação do Templo Dourado"
+Save04 = "O Templo Dourado"
+Save05 = "O Templo Dourado"
+Save06 = "Estação Hidrológica"
+Save07 = "Locais de Procriação da Estação Hidrológica"
+Save08 = "Estação Hidrológica"
+Save09 = "Caverna Oriental"
+Save10 = "Entrada da Fábrica de Robôs"
+Save11 = "Armazém da Fábrica de Robôs"
+Save12 = "Instalação de Mineração"
+Save13 = "Caverna Ocidental"
+Save14 = "Entrada da Torre"
+Save15 = "Exterior da Torre"
+Save16 = "Usina Geo-Térmica"
+Save17 = "Entrada da Distribuição de Energia"
+Save18 = "Instalação de Distribuição Principal"
+Save19 = "Armazém de Instalação da Distribuição"
+Save20 = "Profundezas da Caverna Principal"
+Save21 = "Entrada do Ninho Ômega"
+Save22 = "O Ninho Ômega"
+Save23 = "Saída do Ninho Ômega"
+Save24 = "Laboratório de Genética"
+Save25 = "Thoth GFS"
+Save26 = "Profundezas da Caverna Principal"
+
+[ScoreScreen]
+ClearTime = "LIMPAR TEMPO"
+ItemCollection = "COLEÇÃO DE ITENS"
+SeeYouNextMission = "VEJO VOCÊ NA PRÓXIMA MISSÃO!"
+TheLastMonster = "O ÚLTIMO METROID ESTÁ EM CATIVEIRO, #A GALÁXIA ESTÁ EM PAZ"
+ToBeContinued = "Continua..."
+
+[Notifications]
+GameSaved = "Jogo Salvo"
+EnergyTank = "Tanque de Energia Adquirido"
+MissileTank = "Tanque de Míssil Adquirido"
+SuperMissileTank = "Super Tanque de Míssil Adquirido"
+PowerBombTank = "Tanque da Bomba de Energia Adquirido"
+ScanningBioform = "Escaneando Forma Biológica"
+ScanningMechanism = "Escaneando Mecanismo"
+ScanningEnvironment = "Escaneando o Ambiente"
+
+[IntroCutscene]
+Scene1 = "Metroids. #Numerosos incidentes no passado trouxeram#a luz o poder altamente#destrutivo destas formas de vida espalhando#o medo através de todos os sistemas planetários conhecidos."
+Scene2 = "Esperando assegurar a paz na galáxia,#A Federação Galática ordena o#extermínio da espécie Metroid.#Eu era a melhor candidata para o trabalho."
+Scene3 = "De novo eu me acho sozinha#contra a ameça Metroid.#Que horrores me esperam dentro das profundezas#do mundo natal deles?"
+
+[--------------------Continued Revisions Additions--------------------]
+Ex = "Abaixo estão todos os campos que foram adicionados no arquivo de idioma nas atualizações."

```

This text is not supposed to be translated
```suggestion
Ex = "Below are all the fields that were added to the language file in the updates."
```